### PR TITLE
Integrate with COS

### PR DESCRIPTION
**Pull request by zmraul**
Addresses [DPE-1017](https://warthogs.atlassian.net/browse/DPE-1017)
Add [node_exporter](https://snapcraft.io/node-exporter) snap and a [dashboard](https://grafana.com/grafana/dashboards/1860-node-exporter-full/) to integrate with COS.

Libs and dashboard.json don't need review.

### CODE REVIEW

**Review by deusebio — APPROVED** [review]
Good to go!

File: src/charm.py [hunk 1]
```diff
@@ -44,6 +46,19 @@ def __init__(self, *args):
...
+            jobs=[
+                {
+                    "static_configs": [
+                        {
+                            "targets": ["*:9100"],
+                        }
+                    ]
+                }
+            ],
+        )
```
Comment (marcoppenheimer) [inline src/charm.py:61]: Could we 1 line this and add a lint exception?

File: src/snap.py [hunk 2]
```diff
@@ -35,12 +35,25 @@ def install(self) -> bool:
...
             self.kafka.connect(plug="removable-media")
+
+            node_exporter_plugs = [
+                "hardware-observe",
+                "network-observe",
+                "mount-observe",
+                "system-observe",
+            ]
+            for plug in node_exporter_plugs:
+                node_exporter.connect(plug=plug)
```
Comment (marcoppenheimer) [inline src/snap.py:55]: Ayyyy I did the snap connect! does it work as expected?

**Review by marcoppenheimer — APPROVED** [review]

File: src/snap.py (hunk 2 above)
Comment (zmraul) [reply src/snap.py:55]: Seems that way :)

File: src/charm.py (hunk 1 above)
Comment (zmraul) [reply src/charm.py:61]: I can change this to be better without the exception

**Pull request merged**